src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandler.java

```diff
@@ -106,4 +108,15 @@ private void updateDocument(DwpDocument dwpDocument, DwpResponseDocument dwpResp
         }
     }
 
+    private void setOverrideFields(SscsCaseData sscsCaseData) {
+        OverrideFields overrideFields = sscsCaseData.getSchedulingAndListingFields().getOverrideFields();
+        if (isNull(overrideFields)) {
```

Comment:
Don't we still want to set it if not null?

Reply:
Im not sure about all possibilities, but as I noticed with cases which "returned" from Hearings, they already has this poToAttend field and displaying it.

Reply:
I don't think you should be changing the override values, you should be setting it in the default overidevalues instead. 
The is getDwpIsOfficerAttending determines the default value but doesn't normally get set until a hearing is requested, so you will just be setting it beforehand.

Reply:
Changed to defaultOverrideFields